docs/reference/high-availability/cluster-design.asciidoc

```diff
@@ -60,13 +62,16 @@ To accommodate this, {es} assigns nodes every role by default.
 
 A single node cluster is not resilient. If the the node fails, the cluster will
 stop working. Because there are no replicas in a one-node cluster, you cannot
-store your data redundantly. However, at least one replica is required for a
-<<cluster-health,`green` cluster health status>>. To ensure your cluster can
-report a `green` status, set
-<<dynamic-index-settings,`index.number_of_replicas`>> to `0` on every index. If
-the node fails, you may need to restore an older copy of any lost indices from a
-<<modules-snapshots,snapshot>>. Because they are not resilient to any failures,
-we do not recommend using one-node clusters in production.
+store your data redundantly. However, by default at least one replica is
```

Comment:
Added "by default" otherwise it's not true since you can have a green cluster with no replicas. My bad, I think this must have been an incomplete edit.